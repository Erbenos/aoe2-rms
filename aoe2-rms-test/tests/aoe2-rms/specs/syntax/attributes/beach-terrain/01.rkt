#lang aoe2-rms
; Create a dirt island with beaches that have vegetation.

<LAND-GENERATION>
(base-terrain 'WATER)

<TERRAIN-GENERATION>
(create-terrain 'DIRT
                (number-of-tiles 500)
                (spacing-to-other-terrain-types 1)
                (base-terrain 'WATER)
                (beach-terrain 'DLC_BEACH2))
