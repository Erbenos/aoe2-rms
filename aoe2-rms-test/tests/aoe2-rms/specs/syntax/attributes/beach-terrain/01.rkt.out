<LAND_GENERATION>
base_terrain WATER
<TERRAIN_GENERATION>
create_terrain DIRT {
  number_of_tiles 500
  spacing_to_other_terrain_types 1
  base_terrain WATER
  beach_terrain DLC_BEACH2
}
